server/src/main/java/org/elasticsearch/cluster/node/DiscoveryNodes.java

```diff
@@ -518,8 +518,14 @@ public String shortSummary() {
                     summary.append(", ");
                 }
                 summary.append("removed {");
+                boolean first = true;
                 for (DiscoveryNode node : removedNodes()) {
```

Comment:
I guess `Strings.collectionToCommaDelimitedString()` would do the job?

Reply:
Indeed it would, thanks, see d3928c6.